darshan-util/pydarshan/darshan/backend/api_def_c.py

```diff
@@ -10,8 +10,8 @@
 header = """/* from darshan-logutils.h */
 struct darshan_mnt_info
 {
-    char mnt_type[3031];
-    char mnt_path[3031];
+    char mnt_type[3015];
+    char mnt_path[3015];
```

Comment:
Is this correcting an error unrelated to the primary thrust of this PR or?

Reply:
kind of convoluted, but these array sizes are ultimately based on a macro calculation that is based on the size of the Darshan job structure. (job structure grew by 16 bytes in this PR, and you can see above this reduces these array sizes by 16 bytes).

another argument to somehow automate this, as the only reason i realized this needed to be fixed was by observing weird failures in testing